mappings/net/minecraft/util/SystemDetails.mapping

```diff
@@ -0,0 +1,36 @@
+CLASS net/minecraft/class_6396 net/minecraft/util/SystemDetails
+	FIELD field_33852 MEBIBYTE_DIVIDER J
```

Comment:
Imo `MEBI` suffices. In addition, there is another field which is like `GIGA`, used to convert Hz to GHz. Please map that.